core/src/modules/ops/appRefreshQuery.ts

```diff
@@ -61,4 +61,29 @@ export namespace AppRefreshQueryOps {
       await schedule.enqueueRun();
     }
   }
+
+  export async function test(
+    appRefreshQuery: AppRefreshQuery,
+    refreshQuery?: string
+  ) {
+    let success = false;
+    let message: string;
+    let error;
+
+    if (refreshQuery) appRefreshQuery.refreshQuery = refreshQuery;
```

Comment:
This pattern is a bit dangerous.  The database object is being modified with a temporary value provided by ab optional `refreshQuery`.  This implies that the `AppRefreshQueryOps.runAppQuery()` method should be changed to also allow for an optional refreshQuery to the object doesn't need to be modified.

Reply:
If you then go on to save or update the database object after this method is used, the wrong value for refreshQuery could be saved